plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -320,6 +322,13 @@ void GSSettingsDlg::UpdateRenderers()
 	{
 		GSRendererType ini_renderer = GSRendererType(theApp.GetConfigI("Renderer"));
 		renderer_setting = (ini_renderer == GSRendererType::Undefined) ? GSUtil::GetBestRenderer() : ini_renderer;
+		
```

Comment:
I checked for whitespaces, so I really need to figure out what keeps adding 'em. :/